src/components/AboutUs.js

```diff
@@ -9,16 +9,30 @@ export const AboutUs = () => {
   return (
     <section className=" z-50 relative" id="about-us">
       <div className="about-section relative">
-        <Image src={cityscape} fill className="object-cover sm:object-contain h-[50%] opacity-25 object-bottom" />
+        <Image
+          src={cityscape}
+          fill
+          className="object-cover sm:object-contain h-[50%] opacity-25 object-bottom"
```

Comment:
Add `alt` tag here

Reply:
@8rxn Added "alt" tag in about us section